t/20-function-bqsr_calc.t

```diff
@@ -0,0 +1,159 @@
+use strict;
+use warnings;
+use Test::More tests => 4;
+use Test::Exception;
+use Test::Deep;
+use Test::Warn;
+use File::Temp qw/tempdir/;
+use File::Path qw/make_path/;
+use Cwd qw/cwd abs_path/;
+use Perl6::Slurp;
+use File::Copy;
+use Log::Log4perl qw/:levels/;
+use JSON;
+use Cwd;
+use List::Util qw/first/;
+
+use Moose::Util qw(apply_all_roles);
+
```

Comment:
It should be possible to remove lines 19, 52-56, 127-129 from the test without any adverse effect.

Reply:
done